internal/commands/buildpack_package.go

```diff
@@ -96,6 +96,10 @@ func BuildpackPackage(logger logging.Logger, cfg config.Config, packager Buildpa
 					logger.Warnf("%s is not a valid extension for a packaged buildpack. Packaged buildpacks must have a %s extension", style.Symbol(ext), style.Symbol(client.CNBExtension))
 				}
 			}
+			if flags.Flatten {
+				logger.Warn("Creating a flattened Buildpack package could break the distribution specification. Please use it with caution.")
```

Comment:
```suggestion
				logger.Warn("Flattening a buildpack package could break the distribution specification. Please use it with caution.")
```
(for consistency with error message)